docs/procedures.md

```diff
@@ -77,3 +82,35 @@ We rely on cloud.gov’s [audit tool](https://logs.fr.cloud.gov).  In the event
 
 ## Unsupported System Components Policy
 At present, there are no unsupported components in the SRT system. If currently-used components become deprecated and/or unsupported, we will bump their versions to the most recent stable release. If we're unable to make those bumps, then we will provide justification and document approval for the continued use of the unsupported system components within the relevant GitHub issue.
+
+## GSA Solutions Life Cycle (SLC)
```

Comment:
Did this come from some existing doc or policy? It's very waterfall-like which probably doesn't matter much in the end.

Reply:
I agree. It comes from this:  https://docs.google.com/document/d/1nWx5hPQNF9j31t_vI8yNHGcths_Q-Gs2UkRfZGBdwCM/edit#

To satisfy the control, I took the list of required phases and briefly discussed how we'd perform the activities they list for each one.

The doc recommends a variety of non-waterfally "strategies" that a team can use to move through these phases, with agile being one of them. But the SSP doesn't ask us to declare our strategy, so I figured I'd keep that out of the docs.